soil.php

```diff
@@ -50,6 +50,8 @@ public function set($options) {
   }
 }
 
+require_once(plugin_dir_path(__FILE__) . 'lib/utils.php');
```

Comment:
`plugin_dir_path` is just a wrapper for `dirname(__FILE__)`

Could use `require_once __DIR__ . '/lib/utils.php;`